brain-marks/Localization/en.lproj/Localizable.strings

```diff
@@ -42,7 +42,7 @@
 
 // SettingsView
 "Settings" = "Settings";
-"BrainMarksIsOpenSource..." = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021, work has continued.";
+"BrainMarksIsOpenSource" = "Brain-marks is an open source project! It was created for the Big Brain Hackathon and during Hacktoberfest 2021 and 2022, work has continued.";
 "IfYouWantToContribute" = "If you are interested in contributing yourself, visit the GitHub Repo for more information. Below a list of all contributors can be found. \nYour name could be there!";
```

Comment:
Can you also use the exact text I changed in #133?

Reply:
Changed it accordingly to the issue